assets/javascripts/discourse/initializers/qa-edits.js

```diff
@@ -192,6 +192,8 @@ function initPlugin(api) {
       result.push(qaPost);
     }
 
+    result.push(helper.h("div", `${model.post_number}`));
```

Comment:
This looks like a leftover from local debugging.

Reply:
Good catch :+1: